openmdao/components/meta_model.py

```diff
@@ -56,7 +56,7 @@ def __init__(self):
 
         self._input_size = 0
 
-    def add_param(self, name, val=_NotSet, **kwargs):
+    def add_param(self, name, val=_NotSet, training_data=list(), **kwargs):
```

Comment:
I seem to remember something about an empty list being a bad default value to use.